apollo-composition/src/lib.rs

```diff
@@ -117,15 +117,22 @@ pub trait HybridComposition {
                 self.update_supergraph_sdl(raw_sdl);
                 let satisfiability_result = self.validate_satisfiability().await;
                 self.add_issues(
-                    satisfiability_result_into_issues(satisfiability_result).map(|mut issue| {
-                        for (service_name, connector) in &connectors_by_service_name {
-                            issue.message = issue.message.replace(
-                                service_name.as_str(),
-                                connector.id.subgraph_name.as_str(),
-                            );
-                        }
-                        issue
-                    }),
+                    satisfiability_result_into_issues(satisfiability_result)
+                        .map(|mut issue| {
+                            for (service_name, connector) in &connectors_by_service_name {
+                                issue.message = issue.message.replace(
+                                    service_name.as_str(),
+                                    connector.id.subgraph_name.as_str(),
+                                );
+                            }
+                            issue
+                        })
+                        .chain(once(Issue {
+                            code: "EXPERIMENTAL_FEATURE".to_string(),
+                            message: "Connectors are an experimental feature. Breaking changes are likely to occur in future versions of composition.".to_string(),
```

Comment:
```suggestion
                            message: "Connectors are an experimental feature. Breaking changes are likely to occur in future versions of Apollo Federation.".to_string(),
```

Reply:
I specifically said "composition" because we may make breaking changes (additional validations) in `2.9.1`, which is _not_ a new version of Apollo Federation (the spec).

But this will also come down to us creating greater clarify for our _users_ about what the versions mean in addition to ourselves.